pkg/controllers/capacityreservation/capacitytype/controller.go

```diff
@@ -50,7 +50,7 @@ func NewController(kubeClient client.Client, cp cloudprovider.CloudProvider) *Co
 }
 
 func (*Controller) Name() string {
-	return "nodeclaim.capacityreservation"
+	return "capacityreservation.capacitytype"
```

Comment:
Should this be just `type`?

Reply:
Not really, this controller is responsible for handling fallback from reserved to on-demand capacity when default ODCRs expire. Since we're handling the mutation of the capacity type label, I think this name is appropriate

Reply:
Is it not implied from `capacityreservation`?

Reply:
I don't think so - we're not changing the type of the capacity reservation, we're changing Karpenter's separate concept of a capacity type.